app/controllers/report_controller/reports/editor.rb

```diff
@@ -1722,14 +1722,16 @@ def valid_chargeback_fields
   def check_tabs
     @sb[:miq_tab] = params[:tab]
     active_tab = 'edit_1'
-    case @sb[:miq_tab].split('_')[1]
+    tab_number = @sb[:miq_tab].split('_')[1]
+    tab_title = _("#{TAB_TITLES["edit_#{tab_number}"]} tab is not available until at least 1 field has been selected")
```

Comment:
We must avoid using string math in gettext strings as much as possible (this is to make translation work easier and translations better looking). In this particular case you have to use full strings for each case:
`_('Columns tab is not available ...')` `_('Filter tabs is not available ...')`

Reply:
I removed this commit.